BookingsAPI/Bookings.API/Mappings/SuitabilityAnswersListToResponseMapper.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Bookings.Api.Contract.Responses;
+using Bookings.Domain.Participants;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Bookings.API.Mappings
+{
+    public class SuitabilityAnswersListToResponseMapper
+    {
+        public List<ParticipantSuitabilityAnswerResponse> MapParticipantSuitabilityAnswerResponses(IEnumerable<Participant> participants)
```

Comment:
IEnumerable ?

Reply:
Returning a more derived type (List) provides the API with more operations on the result.